app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -154,8 +152,6 @@ class BrowserTabViewModel(
     private val pixel: Pixel,
     private val dispatchers: DispatcherProvider = DefaultDispatcherProvider(),
     private val userEventsStore: UserEventsStore,
-    private val notificationDao: NotificationDao,
-    private val variantManager: VariantManager,
```

Comment:
I tidy up this two deps as they were unused

Reply:
👍